src/sentry/api/endpoints/organization_traces.py

```diff
@@ -84,53 +88,38 @@ def data_fn(offset: int, limit: int):
                     cast(ParamsType, params),
                     snuba_params=snuba_params,
                     query=serialized.get("query", ""),
-                    selected_columns=["trace", "timestamp"] + serialized["field"],
+                    selected_columns=["trace", "timestamp"],
```

Comment:
I'm guessing this is faster since those columns are ignored (unless they're in the query)?

Reply:
yeah, and this is just to fetch the trace ids, so the less columns we have to read, the faster the query